drizzlepac/haputils/tests/test_processing_utils.py

```diff
@@ -0,0 +1,30 @@
+import os
+import pytest
+import shutil
+from astropy.io import fits
+from drizzlepac.haputils import processing_utils
+
+test_path = "drizzlepac/haputils/tests/"
+
+@pytest.mark.parametrize(
+    "filename",
+    [
+        "sample_ipppssoot_flc.fits", # WFC3/UVIS
+        "sample_ipppssoot_flt.fits", # WFC3/IR
+        "sample_svm_flc.fits", # WFC3/UVIS SVM file
+        "sample_svm_flt.fits", # WFC3/IR SVM file
+        "sample_ipppssoot_flt_w_skycell_header_keyword.fits"
+    ],
```

Comment:
These FITS files are large enough that they should be stored in Artifactory and downloaded when the test is done.

Reply:
The data within the files is truncated to 10 x 10 arrays in each extension to limit the data file size. Do you still think that these should be put on artifactory? I wasn't sure if the goal was to only host files on artifactory even for unit tests.

Reply:
The remaining data is primarily in the headerlets, which I could also remove.

Reply:
Note for deleting headerlets: https://stwcs.readthedocs.io/en/latest/headerlet_tsr/source/hdrlet_tsr.html#delete-headerlet

Reply:
Just an FYI.  Even though Git/GitHub can handle binary files, we need to be circumspect about how many binary (i.e., FITS files in our case) we put into the repository.  When someone downloads the repo, they will have to download potentially large binary files, and this is an inconvenience.  Having said this, I think small files may be fine as long as there are not too many of them.